document/runproperties.go

```diff
@@ -244,3 +244,188 @@ func (r RunProperties) SetVerticalAlignment(v sharedTypes.ST_VerticalAlignRun) {
 		r.x.VertAlign.ValAttr = v
 	}
 }
+
+
+// Bold returns true if run font is bold.
+func (r RunProperties) Bold() bool {
+	x := r.x
+	return getBool(x.B) || getBool(x.BCs)
+}
+
+// Italic returns true if run font is italic.
+func (r RunProperties) Italic() bool {
+	x := r.x
+	return getBool(x.I) || getBool(x.ICs)
+}
+
+// Caps returns true if run font is capitalized.
+func (r RunProperties) Caps() bool {
+	return getBool(r.x.Caps)
+}
+
+// Strike returns true if run is striked.
+func (r RunProperties) Strike() bool {
+	return getBool(r.x.Strike)
+}
+
+// DoubleStrike returns true if run is double striked.
+func (r RunProperties) DoubleStrike() bool {
+	return getBool(r.x.Dstrike)
+}
+
+// Outline returns true if run outline is on.
+func (r RunProperties) Outline() bool {
+	return getBool(r.x.Outline)
+}
+
+// Shadow returns true if run shadow is on.
+func (r RunProperties) Shadow() bool {
+	return getBool(r.x.Shadow)
+}
+
+// Emboss returns true if run emboss is on.
+func (r RunProperties) Emboss() bool {
+	return getBool(r.x.Emboss)
+}
+
+// RightToLeft returns true if run text goes from right to left.
+func (r RunProperties) RightToLeft() bool {
+	return getBool(r.x.Rtl)
+}
+
+// RStyle returns the name of character style.
+// It is defined here http://officeopenxml.com/WPstyleCharStyles.php
+func (r RunProperties) RStyle() string {
+	if r.x.RStyle != nil {
+		return r.x.RStyle.ValAttr
+	}
+	return ""
+}
+
+// Font returns the name of run font family.
+func (r RunProperties) Font() string {
+	if fonts := r.x.RFonts; fonts != nil {
+		if fonts.AsciiAttr != nil {
+			return *fonts.AsciiAttr
+		} else if fonts.HAnsiAttr != nil {
+			return *fonts.HAnsiAttr
+		} else if fonts.CsAttr != nil {
+			return *fonts.CsAttr
+		}
+	}
+	return ""
+}
+
+// EastAsiaFont returns the name of run font family for East Asia.
+func (r RunProperties) EastAsiaFont() string {
+	if fonts := r.x.RFonts; fonts != nil {
+		if fonts.EastAsiaAttr != nil {
+			return *fonts.EastAsiaAttr
+		}
+	}
+	return ""
+}
+
+// HexColor returns the hex value of run color.
+func (r RunProperties) HexColor() string {
+	if color := r.x.Color; color != nil {
+		valAttr := color.ValAttr
+		if valAttr.ST_HexColorRGB != nil {
+			return *valAttr.ST_HexColorRGB
+		}
+	}
+	return ""
+}
+
+// SpacingValue returns the value of run spacing.
+func (r RunProperties) SpacingValue() int64 {
+	if spacing := r.x.Spacing; spacing != nil {
+		valAttr := spacing.ValAttr
+		if valAttr.Int64 != nil {
+			return *valAttr.Int64
+		}
+	}
+	return int64(0)
+}
+
+// SpacingMeasure returns the measure of run spacing.
+func (r RunProperties) SpacingMeasure() string {
+	if spacing := r.x.Spacing; spacing != nil {
+		valAttr := spacing.ValAttr
+		if valAttr.ST_UniversalMeasure != nil {
+			return *valAttr.ST_UniversalMeasure
+		}
+	}
+	return ""
+}
+
+// SizeValue returns the value of run font size.
```

Comment:
// SizeValue returns the value of run font size in points.
Add "in points"

Reply:
fixed